tests/test_install.sh

```diff
@@ -1,192 +1,284 @@
 #!/bin/bash
-
 set -ueo pipefail
 
 if [[ "$#" -ne 1 ]]; then
-    echo "Usage: ./test_install.sh [install | uninstall | upgrade]"
-    exit 1
+  echo "##+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+"
+  echo "##"
+  echo "##  Usage: ./test_install.sh [install | uninstall | upgrade]"
+  echo "##"
+  echo "##+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+"
+  exit 1
 fi
 
 clear
 
+# Github Image Registry PAT
+# $CODESEALER_TOKEN=
```

Comment:
We should explicitly check for this variable being set and warn the caller instead of this.